src/payeedialog.cpp

```diff
@@ -582,6 +609,12 @@ void mmPayeeDialog::CreateControls()
     mmToolTip(m_magicButton, _t("Other tools"));
     tools_sizer2->Add(m_magicButton, g_flagsH);
 
+    m_tbShowAll = new wxToggleButton(buttons_panel, wxID_SELECTALL, _t("Show hidden"), wxDefaultPosition
+        , wxDefaultSize);
+    m_tbShowAll->SetValue(m_showHiddenPayees);
+    mmToolTip(m_tbShowAll, _t("Show hidden payees"));
+    tools_sizer2->Add(m_tbShowAll, g_flagsH);
+
     m_maskTextCtrl = new wxSearchCtrl(buttons_panel, wxID_FIND);
     m_maskTextCtrl->SetFocus();
     tools_sizer2->Prepend(m_maskTextCtrl, g_flagsExpand);
```

Comment:
Can you please add a keyboard accelerator to `_t("Search")` in the line below? Thank you